src/day03/BitRates.php

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types=1);
+
+namespace AdventOfCode\day03;
+
+final class BitRates
+{
+    public function getBitRate(array $input): int
+    {
+        $gama = "";
```

Comment:
`gama` and `epsilon` is forgotten here from the first part of the task. Should be removed.